pyproject.toml

```diff
@@ -34,10 +34,6 @@ dependencies = [
     "brainbuilder",
     # MORPHOLOGY CONTAINERIZATION
     "bluepysnap",
-    # FAST API SERVICE
-    "fastapi",
-    "uvicorn",
-    "starlette",
     # JUPYTER
     "jupyter",
     "notebook",
```

Comment:
Commenting here, but related to `tool.uv.sources` below: do bluepysnap and brainbuilder still need to be installed from git, instead of using a package?